pages/common/mongorestore.md

```diff
@@ -0,0 +1,19 @@
+# mongorestore
+
+> Utility to import a collection or database from a binary dump into a MongoDB instance.
+
+- Import a bson data dump from a folder to a mongodb instance:
+
+`mongorestore --db {{database_name}} {{path/to/folder}}`
+
+- Import a bson data dump from a folder to a given mongodb database host running at a given port with user authentication; user will be prompted for password:
+
+`mongorestore --host {{database_host:port}}  --db {{database_name}} --username {{username}} {{path/to/folder}} --password`
```

Comment:
If I'm reading you correctly, by specifying the host and the port, the --db option should be redundant, right? Could one say it works as a check to ensure one's connecting to the desired database and not another that happens to be hosted in the same host and port? I mean, what happens if you try to connect to a db giving the correct host address, port, and authentication, but a mismatching db name? Does the command fail?

And in the opposite case: if the port isn't provided in the command, and say the database isn't hosted in the default port that the command uses to connect; I suppose the --db parameter is not sufficient to make it work, right?

If both these assumptions are correct, then the --db option is redundant at best, and useless at worst, so I'd suggest omitting it since the command is quite long already. Otherwise, please correct any misconceptions on my part :)